packages/block-library/src/image/index.php

```diff
@@ -33,17 +33,10 @@ function render_block_core_image( $attributes, $content, $block ) {
 
 	$should_load_view_script = false;
 	$link_destination        = isset( $attributes['linkDestination'] ) ? $attributes['linkDestination'] : 'none';
+
 	// Get the lightbox setting from the block attributes.
-	if ( isset( $attributes['behaviors']['lightbox'] ) ) {
-		$lightbox_settings = $attributes['behaviors']['lightbox'];
-		// If the lightbox setting is not set in the block attributes, get it from the theme.json file.
-	} else {
-		$theme_data = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data()->get_data();
-		if ( isset( $theme_data['behaviors']['blocks'][ $block->name ]['lightbox'] ) ) {
-			$lightbox_settings = $theme_data['behaviors']['blocks'][ $block->name ]['lightbox'];
-		} else {
-			$lightbox_settings = null;
-		}
+	if ( isset( $attributes['lightbox'] ) ) {
+		$lightbox_settings = $attributes['lightbox'];
```

Comment:
I'm assuming that this will be the structure of the "new" attribute that controls the lightbox.